src/GIRUtils.php

```diff
@@ -160,30 +190,12 @@ public static function getOpenIdToken() {
     }
   }
 
-  /**
-   * Get all employments with engagements in the specified organisation unit.
-   *
-   * NOTE: Do not recurse into children.
-   */
-  public static function getEmployees($org_unit_uuid) {
```

Comment:
Jeg går ud fra denne er slettet, fordi den ikke bliver brugt nogle steder?

Reply:
Nemlig! Der er ingen grund til at slæbe rundt på den, hvis nu vi ikke får brug for den. Vi kan let genskabe den eller fiske den ud af git-historikken.